docs/FeatureManagement/Samples/TargetingFilter.modified.sample.json

```diff
@@ -0,0 +1,60 @@
+{
+    "feature_management": {
+        "feature_flags": [
+            {
+                "id": "ComplexTargeting",
```

Comment:
This flag is a duplicate and not used as far as I can tell.

Reply:
The idea is that we run these same tests but slightly changed % values, we can add at test for this bumping up, but haven't gotten around to it yet. All of this is sort of the fact that this didn't start out like this.